pkg/config/release.go

```diff
@@ -20,6 +20,8 @@ const (
 	CiopConfigInRepoPath = "ci-operator/config"
 	// TemplatesPath is the path of the templates from release repo
 	TemplatesPath = "ci-operator/templates"
+	// Name of the configmap that stores all ci-operator configs
+	CiOperatorConfigsCMName = "ci-operator-configs"
```

Comment:
This is not true -- we are sharded. See the `ConfigMapName()` method on the `config.Info` struct

Reply:
I'm aware of that, I want to tackle that tomorrow. IMHO it's worth a separate PR. I only moved this constant, not created it. Rehearse is a little broken after the sharding anyways.

Reply:
Addressed in https://github.com/openshift/ci-operator-prowgen/pull/104